package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "katex",
-  "version": "0.13.0",
+  "version": "0.0.0-development",
```

Comment:
Is the plan to only use real versions for release versions?  Should we also be changing the version back to this after a release is completed?

Reply:
Yes, this will be only updated in released versions, which are uploaded to `npm`.